fix(docs): Small edits

justinrlee opened the pull request.

mhumeSF committed:
Syntax edit

mhumeSF committed:
Correct flag

justinrlee committed:
chore(docs): Add instructions to create service account manually (#663)

brian-armory commented:
configure_ingress got renamed to _spinnaker/exposing_spinnaker.md so that it matches the page title.

justinrlee committed:
Update install-on-gke.md (#665)

317brian committed:
docs(fix): remove Jira integration since it is currently not in the product (#668)

Commit:
chore(release): notes for Armory Spinnaker 2.16.1 (#669)

317brian committed:
docs(dinghy): add required hal version for gitlab (#670)

317brian committed:
docs(airgap): update nesting and content (#671)

Made updates to the air-gapped environments doc that Nicolas put out.

justinrlee committed:
docs(chore): clarify what's needed for slack notifications (#673)

317brian committed:
docs(overview): incorporate comments from reviewers (#666)

* docs(overview): incorporate comments from reviewers

* docs(updates): add more feedback from Brandon

* docs(fix): drop red black and use only blue green

ncknt committed:
fix(doc): fix Clouddriver SQL setup (#674)

317brian committed:
docs(overview): make updates to the overview page (#675)

kevinawoo committed:
refact(integration_test_pipeline): readability and dedeprecation (#685)

The depreciation statement on this page was inaccurate

Commit:
chore(api/endpoints): small edits for api endpoints (#684)

kevinawoo committed:
refactor(rate-limits): remove unpublished page (redirected) (#686)

https://github.com/armory/documentation/blob/master/_spinnaker_install_admin_guides/rate-limit.md handles this page

kevinawoo committed:
refact(config): remove older ec2 install config page (#683)

Commit:
chore(kubernetes/accounts): small edits to kubernetes accounts d… (#682)

kevinawoo committed:
fix(ecr): add version number (#679)

german-muzquiz committed:
doc(operator): Typos and minor changes (#676)

kevinawoo committed:
fix(ecr): make words better (#681)

kevinawoo committed:
feat(ecr): add codeblock syntax highlighting (#680)

Commit:
chore(aws/accounts): small edits to aws docs (#678)

german-muzquiz committed:
doc(operator): Typos and minor changes (#677)

Commit:
chore(releases): notes for Armory Spinnaker 2.16.2 (#692)

karlomendozaarmory committed:
chore(doc): doc for static baseline judge (#694)

kevinawoo committed:
refact(rate-limits): make readable words and update docs (#687)

Commit:
fix(admin/s3): small typo fix (#689)

justinrlee committed:
Update add-aws-account-iam.md (#695)

justinrlee committed:
docs(fix): Add missing step for creating pipeline from pre-exist… (#696)

317brian committed:
Fix typo in Deployment Strategies/Canary (#697) (#698)

Signed-off-by: Aimee Ukasick <aimeeu.opensource@gmail.com>

Commit:
chore(gitops): add languages to codeblocks (#688)

Commit:
fix(admin/ratelimits): some wording changes (#690)

rnevius committed:
feat(site): Redirect /install-guide/gettings_started/ (#558)

Redirects to /spinnaker/install/
Removes references to the original gettings_started page

317brian committed:
docs(edits): update policy engine doc (#672)

KathrynLewis committed:
feat(docs): add vault config to operator docs and vice versa (#699)

* feat(docs): add vault config to operator docs and vice versa

* copyedits

* remove limitations section


Co-authored-by: null <brian.le@armory.io>

317brian committed:
docs(policyEngine): update configMaps and cleanup (#700)

* docs(policyEngine): update configMaps to match OPA docs examples

* docs(policyEngine): fix hal command

* add toc

* incorporate feedback

317brian committed:
docs(policyengine): fix markdown formatting (#701)

away168 committed:
fix(doc): Adding halyard as pod instructions (#702)

* Adding halyard as pod instructions

* adding more details to kubectl command

317brian committed:
docs(policy_engine): formatting fixes (#703)

* docs(policy_engine): formatting fixes

* put back in original yaml because trying to make yaml look nice in markdown makes brian sad

* add in more context and PEBKAC help

Commit:
chore(release): notes for 2.16.3 (#705)

german-muzquiz committed:
doc(operator): Added instructions for uninstalling operator (#704)

* doc(operator): Added instructions for uninstalling operator

* doc(operator): Uninstall redaction

Commit:
docs(documentation): Add content for Minnaker (#706)

karlomendozaarmory committed:
add clarification for orca mysql migration on dual run (#708)

Commit:
chore(release): document release of 2.17.0 (#709)

* chore(release): document release of 2.17.0

* chore(releases): map OSS 1.17 to 2.17

317brian committed:
chore(toc): move toc to top (#710)

317brian committed:
doc(fix): docs-39 add missing closing apostrophe (#711)

Commit:
chore(release): mention plugins in 2.17.0 release notes (#712)

away168 committed:
fix sumologic echo-local folder (#714)

Commit:
chore(releaes): notes for halyard-armory 1.8.0 (#716)

Commit:
chore(release): release notes for 2.17.1 and known issue (#717)

KathrynLewis committed:
fix(docs/api): update url for swagger ui (#718)

* fix(docs/api): update url for swagger ui

* typo

* typo

* more edits

* make port optional

justinrlee committed:
chore(docs): Update TF docs with supported TF versions (#720)

jhindulak committed:
docs(fix): Minor documentation fixes (#715)

* Updating links to proper URL from localhost:4000
* Spelling error on 'Bitbucket' in Terraform documentation

317brian committed:
docs(fix): rename files to fix broken imgs (#722)

317brian committed:
doc(copyedit): remove lines sales doesn't like (#707)

* remove lines sales doesnt like

* remove lines sales doesnt like

317brian committed:
docs(fix): copyedit plugin docs (#693)

* docs(plugins): copyedit plugin creators doc

* docs(fix): copyedit plugin users doc

* make changes from review

317brian committed:
docs(fix): remove notification example (#724)

ryanpei committed:
fix(docs): Small edits (#723)

317brian committed:
docs(version): add mysql version (#726)

Commit:
chore(release): notes for armory spinnaker 2.17.2 (#727)

* chore(release): notes for armory spinnaker 2.17.2

* fix(notes): link to 1.17.5 release notes of OSS

german-muzquiz committed:
chore(release): 2.17.2 spinnaker community highlights (#728)

317brian committed:
docs(version): add recommended Spinnaker versions (#729)

* docs(versions): add min spinnaker version

* docs(versions): add min spinnaker version

* Update _spinnaker/policy_engine.md

Co-Authored-By: Mark Rampton <mark.rampton@armory.io>

dependabot[bot] committed:
chore(deps-dev): bump rack from 1.6.11 to 1.6.12 (#732)

Bumps [rack](https://github.com/rack/rack) from 1.6.11 to 1.6.12.
- [Release notes](https://github.com/rack/rack/releases)
- [Changelog](https://github.com/rack/rack/blob/master/CHANGELOG.md)
- [Commits](https://github.com/rack/rack/compare/1.6.11...1.6.12)

Signed-off-by: dependabot[bot] <support@github.com>

317brian committed:
docs(fix): fix broken links (#733)

* Update kayenta.md

* Update kayenta.md

* Update kayenta.md

* Update kayenta.md

aimeeu committed:
Add google custom search script and div (#735)

mhumeSF committed:
Dedupe (#731)

Commit:
chore(release): notes for armory-halyard 1.8.1 (OSS 1.28.0) (#737)

317brian committed:
doc(relnotes): update permissions (#738)

Commit:
chore(plugins): updated frontend creators docs (#736)

* chore(plugins): updated frontend creators docs

* make minor copyedits

* fix(plugins): fixing up phrasing and grammar

Co-authored-by: brian-armory <53799971+brian-armory@users.noreply.github.com>

317brian committed:
docs(chore): update docs to match github docs (#730)

* docs(update): update docs to match github docs

* fix bolding

* minor edit

* minor edit

* add missing periods

* add incompatible change

* docs(operator): update the operator docs

* address comments

aimeeu commented:
@justinrlee Please rebase so we can merge this PR. Thanks!

jorgebee65 committed:
docs(kustomize): add 2.17( OSS 1.17) docs using git/repo guide (#739)

* update guide about how to build and deploy a manifest using Kustomize 1.17

317brian committed:
docs(accountmgmt): add account mgmt docs (#734)

* docs(accountmgmt): add account mgmt docs

* wip account mgmt comments

* docs(account_mgmt): incorporate feedback

MichaelDBrown committed:
docs(troubleshooting): Add note about waiting until containers a… (#741)

* Wait until containers are running before expecting to connect

* minor copyedits

Co-authored-by: brian-armory <53799971+brian-armory@users.noreply.github.com>

317brian committed:
docs(terraformer): add line about creating the directory beforeh… (#743)

317brian committed:
docs(demo): remove link to demo env (#742)

Commit:
chore(release): notes for Armory Spinnaker 2.17.3 (#744)

justinrlee committed:
fix(docs): Add missing backticks (#745)
